src/Extensions/Logging/InitStaticLogger.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// InitStaticLogger is the logger to be used before the uls logger is set
+	/// not to be used as main logger
+	/// </summary>
+	public sealed class InitStaticLogger
+	{
+		private static ILogger s_logger = LoggerFactory.Create(builder =>
```

Comment:
We would probably need to make logger field publicly (like `Instance` property) so people could use it before generic host created, for example in configuration provider. Alternatively, we could have another method to report those events

Reply:
Actually, what do you think about this API
```CSharp
public static InitializationLogger
{
     static void InitializationSucceded(string serviceName)
     static void InitializationEvent(string message);
     static void InitializationFailure(Exception? exception = null, string message = "");
}
```

Reply:
Thinks its good the only change I would make though is add optional message param for succeed